src/main/java/gr/grnet/pccapi/endpoint/ProviderEndpoint.java

```diff
@@ -18,6 +25,13 @@ public ProviderEndpoint(ProviderService providerService) {
         this.providerService = providerService;
     }
 
+    @Tag(name="Providers")
+    @APIResponse(
+            responseCode = "200",
+            description = "The list of available providers",
```

Comment:
Α Provider is an organisation that hosts the handle service. From here you may get the list of all the available providers in PCC-api.